src/tools/wallmap/wallmap.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/fogleman/gg"
+	"golang.org/x/image/bmp"
```

Comment:
@aurelien-rainone I'm duploduplo. Better to untag people that may be not interested in the project :D